nistats/version.py

```diff
@@ -44,7 +44,12 @@
         'install_info': _NISTATS_INSTALL_MSG}),
     ('nibabel', {
         'min_version': '1.1.0',
-        'required_at_installation': False}))
+        'required_at_installation': False}),
+    ('pandas', {
+        'min_version': '0.9.0',
```

Comment:
Just using this as a reminder here to set the pandas version consistently here too.